ui/lib/kv/addon/components/page/configuration.hbs

```diff
@@ -0,0 +1,12 @@
+<KvPageHeader @breadcrumbs={{@breadcrumbs}} @model={{@model}} @isEngineView="true">
```

Comment:
This is a nit and since you're just looking if `isEngineView` has value it won't impact the outcome but it's a bit strange to pass a boolean as a string rather than `@isEngineView={{true}}`

Reply:
Good catch. I didn't think that one through.